proposals/007-kroxylicious-1.0.md

```diff
@@ -0,0 +1,96 @@
+# Kroxylicious 1.0 and patch releases
+
+This proposal is for a Kroxylicious 1.0 release, and an improved commitment to making patch releases.
+
+## Current situation
+
+Kroxylicious already uses [Semantic Versioning][semver], but so far all releases have been `0.x.y` versions.
+
+The project currently promises API compatibilty covering at "least 3 releases, and at least 3 months".
+
+So far, we have done patch releases (i.e. releases like `0.x.1`) only when a new minor release with a fix is not already released, and not imminent.
+
+## Motivation
+
+Under semantic versioning, releases with a major number of 0 are allowed to break compatibility with previous releases.
+The software released by project is maturing, and existing users would benefit from having a stronger guarantee about when compatibility between releases might be broken.
+
+At this time there are already a number of users using Kroxylicious in production, however, some potential users might be put off from adopting Kroxylicious by the `0.` version number. 
+A 1.0 release would be a signal that we believe Kroxylicious is production-ready.
+
+## Proposal
+
+### What will have a compatibility guarantee?
+
+We propose that the release that was going to be called `0.18.0` instead be called `1.0`.
+By implication of Semantic Versioning, that means we would be committing to not breaking compatibilty until some future `2.0` release.
+The time of such a `2.0` release is currently neither known nor planned.
```

Comment:
One of the caveats expressed internally was we want to be clear that we will make a 2.0 release,  and while I don't disagree with the point, I think its diluting that intent. 

As a starter for 10:
```suggestion
The time of such a `2.0` release is currently neither known nor planned, however we expect to be releasing major versions anually.
```